1.Strings/stringtask14.py

```diff
@@ -0,0 +1,12 @@
+"""Write a Python program that accepts a comma-separated sequence of words
+as input and prints the distinct words in sorted form (alphanumerically).
+Sample Words : red, white, black, red, green, black
+Expected Result : black, green, red, white,red
```

Comment:
Wrong Logic, check this example: "red, black, red,red, red, pink, green, pink, pink", (didn't accept spaces inside list object)